### ISSUE TITLE
JSON parse error from Diplomat::Status.leader()

### ISSUE THREAD

**Issue by quulah** [WeAreFarmGeek/diplomat#153]
I have a single test server. When using Diplomat::Status.leader(), it raises an exception.

Consul 0.7.0
Diplomat 2.0.1

```
$ curl -XGET http://localhost:8500/v1/status/leader
"10.0.2.15:8300"
```

```
irb(main):001:0> require 'diplomat'
=> true
irb(main):007:0> Diplomat::Status.leader()
JSON::ParserError: 784: unexpected token at '"10.0.2.15:8300"'
	from /usr/lib/ruby/2.3.0/json/common.rb:156:in `parse'
	from /usr/lib/ruby/2.3.0/json/common.rb:156:in `parse'
	from /var/lib/gems/2.3.0/gems/diplomat-2.0.1/lib/diplomat/status.rb:11:in `leader'
	from /var/lib/gems/2.3.0/gems/diplomat-2.0.1/lib/diplomat/rest_client.rb:43:in `method_missing'
	from (irb):7
	from /usr/bin/irb:11:in `<main>'
```

**Comment by quulah** [comment]
.peers() seems to work, it returns an array and that is correctly parsed as JSON.

The single string seems to be problematic.

**Comment by taharah** [comment]
This has been fixed with the release of v2.0.2.

**Comment by mkarklins** [comment]
Here is a failing [gist](https://gist.github.com/mkarklins/1686af1636b54c4ceceaccde2d247e61), produces `parse': 784: unexpected token at '"127.0.0.1:8300"' (JSON::ParserError)`

JSON 1.8.6 doesn't seem to parse correctly the response. Upgrading to the latest `json` fixes the problem.

Not sure if this should be added to gemspec, but an FYI for those who may experience this issue on `>= 2.0.2` branch.
